src/gameState.ts

```diff
@@ -0,0 +1,82 @@
+import { Board } from "./board/board";
+import { loadGameConfigFromFile } from "./gameLoader";
+import { Coordinates, checkIfMoveIsAllowed } from "./move/move";
+import { Player } from "./player";
+import { Tile } from "./tile/tile";
+
+const initialGameState: GameState = {
+  currentPlayer: "black",
+  turnNumber: 1,
+};
+
+export interface GameState {
+  board?: Board;
+  turnNumber: number;
+  currentPlayer: Player;
+  winner?: Player;
+  selectedMoveCordinates?: Coordinates;
+}
+export type GameActionName =
+  | "SELECT_TILE"
+  | "FORFEIT"
+  | "WAITING"
+  | "END_TURN"
+  | "LOAD_BOARD";
+export interface GameStateAction {
+  name: GameActionName;
+  selectedMoveCoordinates?: Coordinates;
+  newBoard?: Board;
+}
+
+const setAllowedTilesOnBoard = (gameState: GameState): Board => {
+  if (gameState.board == null) {
+    return null;
+  }
+
+  if (gameState.turnNumber === 1) {
```

Comment:
You are checking that `gameState.turnNumber === 1` twice, here and at line 48. Is this intended?

Reply:
No longer exists since merge with https://github.com/marmelab/kamon/pull/8